unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/IBackpackFiltersComponentView.cs

```diff
@@ -0,0 +1,17 @@
+﻿using DCLServices.WearablesCatalogService;
+using System;
+using System.Collections.Generic;
+
+namespace DCL.Backpack
+{
+    public interface IBackpackFiltersComponentView
+    {
+        event Action<bool> OnOnlyCollectiblesChanged;
+        event Action<HashSet<string>> OnCollectionChanged;
+        event Action<(NftOrderByOperation type, bool directionAscendent)> OnSortByChanged;
+        event Action<string> OnSearchTextChanged;
+
+        void Dispose();
```

Comment:
Why are we creating a Dispose in the interface? Shouldn't we use in the BackpackFiltersComponentView the BaseComponent interface that already has the Disposal method?